apheleia-formatters.el

```diff
@@ -35,6 +35,7 @@
               "-"))
     (brittany . ("brittany"))
     (buildifier . ("buildifier"))
+    (biome . ("apheleia-npx" "biome" "format" "--stdin-file-path" filepath))
```

Comment:
FYI - `format` does not sort imports (for some strange reason), however applying both `check` and `--fix` to the command then it does.